saw-central/src/SAWCentral/Value.hs

```diff
@@ -598,6 +602,11 @@ data TopLevelRO =
 
   }
 
+-- | Current state of the Java sub-system.
+data JavaCodeBase =
+    JavaUnitialized                 -- ^ Not yet initialized
+  | JavaInitialized JSS.Codebase    -- ^ Initialized
```

Comment:
These Haddock comments don't really do much other than restate the name of the data constructors. How about something a bit more descriptive, like this:

```suggestion
    JavaUnitialized
    -- ^ No Java-related commands have been invoked yet.
  | JavaInitialized JSS.Codebase
    -- ^ At least one Java-related command has been invoked successfully.
    -- We cache the resulting 'JSS.Codebase' for subsequent commands.
```